### PR TITLE
Fix Helm templates for Compass managed GCP PostgreSQL

### PR DESCRIPTION
**Pull request by life0215**
**Description**

Changes proposed in this pull request:

- Add cloudsql-proxy container to compass-migration job
- Add missing volumemounts to compass-director deploy template

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/6711
compass/director chart deployment template missing volume mounts in case of managed GCP database
https://github.com/kyma-project/kyma/issues/6713
Compass migrator-job template not handling case of managedGCP PostgreSQL

### CODE REVIEW

File: resources/compass/templates/migrator-job.yaml [hunk 1]
```diff
@@ -18,6 +18,19 @@ spec:
...
             restartPolicy: Never
             containers:
+                {{if eq .Values.global.database.embedded.enabled false}}
+                - name: cloudsql-proxy
+                  image: gcr.io/cloudsql-docker/gce-proxy:1.11
+                  command:
+                  - /bin/sh
+                  args:
+                  - -c
+                  - '/cloud_sql_proxy -instances={{ .Values.global.database.managedGCP.instanceConnectionName }}=tcp:5432 -credential_file=/secrets/cloudsql-instance-credentials/credentials.json & sleep 30; exit'
```
Comment (pkosiec) [inline resources/compass/templates/migrator-job.yaml]: Why do we need to sleep for 30 seconds in parallel?

Comment (life0215) [reply resources/compass/templates/migrator-job.yaml]: This way, this container will exit running status after 30 seconds,  won't block the job mark itself completed.

Comment (pkosiec) [reply resources/compass/templates/migrator-job.yaml]: What if the schema migration takes longer than 30 seconds? The proxy will exit and then the migrator will fail - do I understand correctly?

If so, the solution doesn't seem to be right. Can we do it the same way as killing Istio sidecar? Please see my other comment.

File: resources/compass/templates/migrator-job.yaml [hunk 2]
```diff
@@ -60,3 +73,9 @@ spec:
                   args:
                     - "-c"
                     - "./run.sh; exit_code=$?; echo '# KILLING PILOT-AGENT #'; curl -XPOST http://127.0.0.1:15020/quitquitquit; sleep 5; exit $exit_code;"
```
Comment (pkosiec) [inline resources/compass/templates/migrator-job.yaml]: If `cloud_sql_proxy` doesn't have a dedicated endpoint, please use the `pkill -INT` command like we did before `quitquitquit` endpoint for Istio sidecar: https://github.com/kyma-project/kyma/pull/6343/files#diff-44b7cd4a24bf84a8d012823ad10723a6L19

File: resources/compass/templates/migrator-job.yaml (hunk 1 above)
Comment (life0215) [reply resources/compass/templates/migrator-job.yaml]: Thanks for the suggestion, has updated this PR with an workaround.

File: resources/compass/templates/migrator-job.yaml (hunk 2 above)
Comment (life0215) [reply resources/compass/templates/migrator-job.yaml]: Has updated the solution again using pkill, gcr.io/cloudsql-docker/gce-proxy:1.11 is not working with pkill -INT well, using gcr.io/cloudsql-docker/gce-proxy:1.13

**Pull request merged**